### DISCUSSION TITLE
Custom domain not working

### DISCUSSION THREAD

**Discussion by PranavArya37** [vercel/community#5312] [Help] [answered]
### Summary

I've set up the custom domain on Vercel, and the build has been successfully deployed. The preview link from Vercel can display the static website. I've verified all DNS settings using a checker, and both A record and CNAME records are pointing to Vercel. However, the website remains inaccessible.

While accessing domain name some random ads are coming like below-
![image](https://github.com/vercel/community/assets/43077165/851f873d-f709-46cb-8d09-6997cbb3e041)

### Steps to Reproduce

1. Custom domain setup as mentioned
2. Open URL

**Comment by robinmuhia** [comment]
Yours works?? What did you do. Mine has been not working for two days now since i changed the nameservers.

**Reply by PranavArya37** [reply, marked as answer]
Mine is working now
